core/src/test/scala/precog/AutoBumpSpec.scala

```diff
@@ -198,22 +198,22 @@ class AutoBumpSpec(params: CommandLine) extends Specification with ScalaCheck wi
 
       val result = tryUpdate.runA(env).unsafeRunSync()
 
-      result must beRight(matchA[UpdateBranch]
-          .label(ChangeLabel.Breaking)
+      result must matchA[UpdateBranch]
+          .label(Some(ChangeLabel.Breaking))
           .changes(List(
             "Updated **revision** org-repo-1 `1.0.0` → `1.0.1`",
             "Updated **feature** org-repo-2 `2.0.1` → `2.1.0`",
-            "Updated **breaking** org-repo-3 `3.1.2` → `4.1.1`")))
+            "Updated **breaking** org-repo-3 `3.1.2` → `4.1.1`"))
     }
 
     "not fail if dependencies have already been updated" in {
       implicit val runner: Runner[Test] = noUpdatesRunner
       val env = TestEnv.empty
       val tryUpdate = autobump.tryUpdateDependencies[Test](Runner.DefaultConfig)
 
-      val result = tryUpdate.runA(env).unsafeRunSync()
+      val _ = tryUpdate.runA(env).unsafeRunSync()
 
-      result must beLeft(NoLabel)
+      ok    // ?
```

Comment:
I'd think it should become `result must beNone`.

Reply:
No, wait, `tryUpdateDependencies` is still returning an `Either`, but the change label isn't being mapped as a result, so it is always right. So the check would be to verify that the `UpdateBranch` returned inside that `Right` has a `None` as change label.

Reply:
And I see the return type has changed to remove the `Either`. Yeah, I think `result.label must beNone`.

Reply:
It's not `None` though.

Reply:
If it is not `None`, it will break detection of no changes.

Reply:
In fact... maybe it will break it anyway. Ok, I gotta think this through more carefully.

Reply:
I don't think so. `tryUpdateDependencies` returns `UpdateBranch` now. It previously returned `Either[Warnings, UpdateBranch]`, but when I made label detection optional, that removed the *only* warning it could generate. If detection of no-changes was relying on this code, it was broken to begin with.

Reply:
Oh, you mean the label. Alright that's a fair assertion. :-)

Reply:
Looks like the case where a `Warning` would be returned never happened under normal circumstances, because a change label was always printed, defaulting to revision, so nothing got broken.

On the other hand, the runner used doesn't print anything so it _should_ be `None` here. Does it fail if you test for `None`?